### PR TITLE
KH-341: change the page label from 'Cervical Cancer' to 'Cancer' in CC Screening form

### PR DESCRIPTION
**Pull request by kazlaw**
Link to ticket [here](https://mekomsolutions.atlassian.net/browse/KH-341)

### CODE REVIEW

**Comment by kazlaw** [comment]
cancer-cancer

**Comment by kazlaw** [comment]
cancer-cancer

File: configs/openmrs_config/ampathformstranslations/cervical_cancer_screening_km.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
       "Menstrual History": "ប្រវត្តិនៃការមករដូវ",
       "Menstrual Bleeding History": "ប្រវត្តិនៃការធ្លាក់ឈាមរដូវ",
       "Risk Factors" :"កត្តាហានិភ័យ",
-      "Cervical Cancer": "ជំងឺមហារីកមាត់ស្បូន",
+      "Cancer": "Cancer",
```
Comment (kdaud) [inline configs/openmrs_config/ampathformstranslations/cervical_cancer_screening_km.json:12]: @kazlaw should we consider adding the translation in the followup commit?

Comment (kazlaw) [reply configs/openmrs_config/ampathformstranslations/cervical_cancer_screening_km.json:12]: Hi @kdaud  all the tickets currently have pending translations. I will assign them to you as a reviewer when they are ready. The progress is reported on the ticket comments in Jira for now

Comment (kazlaw) [reply configs/openmrs_config/ampathformstranslations/cervical_cancer_screening_km.json]: The Khmer translation was added 

cc-tab

Comment (kazlaw) [reply configs/openmrs_config/ampathformstranslations/cervical_cancer_screening_km.json]: cctab-big

**Pull request merged**